app/controllers/userAuth.js

```diff
@@ -0,0 +1,40 @@
+var jwt = require('jsonwebtoken');
+var mongoose = require('mongoose');
+var User = mongoose.model('User');
+var secret = 'ttv7yttvytftfftytftt';
```

Comment:
### 🔴  Critical
The secret is used to encrypt the user object and should not be added to the code rather retrieved from the environment variable.

```js
var secret = process.env.SECRET_TOKEN_KEY;
```